### PR TITLE
feat: transaction page implemented

### PR DESCRIPTION
**Pull request by aulneau**

### CODE REVIEW

File: src/components/rows.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    currency: string;
+  };
+  block: number;
+  post_conditions: string;
+  sponsored: boolean;
+  state: string;
+  timestamp: number;
+}
+
+interface TxSchema {
```
Comment (kyranjamie) [inline src/components/rows.tsx]: repeated type

File: src/components/rows.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import { Caption } from '@components/typography';
+import { Card } from '@components/card';
+import { toSnakeCase } from '@common/utils';
+
+interface TxSchema {
+  txid: string;
+  sender: string;
+  recipient: string;
+  type: number;
+  fees: {
```
Comment (kyranjamie) [inline src/components/rows.tsx]: Better to make a separate type for Fees above, so it doesn't need to be repeated on line :44

**Comment by kyranjamie** [comment]
@zone117xs PR on sidecar returns data we can hook up now, so will be great to get this PR merged, even if not totally ready

File: src/components/rows.tsx (hunk 2 above)
Comment (kyranjamie) [reply src/components/rows.tsx]: These can go now, see models folder

**Comment by aulneau** [comment]
Rodger, sounds good. I will push and fix any conflicts, and we can merge today @kyranjamie

**Pull request merged**